tfjs-backend-webgpu/src/backend_webgpu_test.ts

```diff
@@ -252,8 +252,8 @@ describeWebGPU('backend webgpu', () => {
       tf.setBackend('webgpu');
       const d = tf.add(a, b);
       await d.data();
+      expect(() => d.dataSync()).not.toThrow();
     };
-
-    expect(f).not.toThrow();
```

Comment:
Not sure for async function, how should we use expect.not.toThrow. So I changed the logic here and move it to the body of async function.